### PR TITLE
Move unit-tests/ to tests/

### PR DESCRIPTION
**Pull request by SidRoberts**

### CODE REVIEW

File: tests/unit/Mvc/Router/AnnotationsTest.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Phalcon\Test\Unit\Mvc\Router;
+
+use Phalcon\Test\Module\UnitTest;
+
+require_once TESTS_PATH . '/../unit-tests/controllers/AnnotationController.php';
```
Comment (sergeyklay) [inline tests/unit/Mvc/Router/AnnotationsTest.php]: This file should be moved to te `tests/_data/controllers`. Also take a look at `tests/_config/bootstrap.php` (`$loader->registerNamespaces`)

**Comment by sergeyklay** [comment]
@SidRoberts Ready to merge?

**Comment by SidRoberts** [comment]
Yeah, it's ready to merge.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**